central/complianceoperator/v2/report/manager/complianceReportgenerator/report_gen_impl.go

```diff
@@ -316,6 +325,11 @@ func (rg *complianceReportGeneratorImpl) sendEmail(ctx context.Context, zipData
 		}
 		return
 	}
+
+	if !features.ScanScheduleReportJobs.Enabled() {
+		return
+	}
```

Comment:
That place for early return feels a bit strange. If we disable this flag, we are still sending an email, so why we don't want to persist the successful status of that in the DB? We are persisting errors, so maybe we should persist success too?
Could you please double-check that?

Reply:
Indeed it is also checked inside `reportUtils.UpdateSnapshotOnError`.